spi/topology/src/main/java/quarks/topology/spi/graph/TWindowImpl.java

```diff
@@ -30,4 +35,20 @@
         Aggregate<T,U,K> op = new Aggregate<T,U,K>(window, processor);
         return feeder().pipe(op); 
     }
+
+    @Override
+    public <U> TStream<U> batch(BiFunction<List<T>, K, U> batcher) {
+        batcher = Functions.synchronizedBiFunction(batcher);
+        Window<T, K, List<T>> window =
+                Windows.window(
```

Comment:
The primary difference (from the current implementation)  being that the trigger policy  calls evict instead of the contents policy.